crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -113,7 +113,7 @@ def compute_status
           )
         end
         nova = NodeObject.find("roles:nova-controller").first
-        ret[:no_live_migration] = true unless nova["nova"]["use_migration"]
```

Comment:
If we return early there would be no indication at all that something is missing and technically if there's no nova at all, live migration is not configured in nova for sure :)

The real question is: should there be a separate error code for not having nova at all or can we rely on the one for live migration?